### PR TITLE
[bug] fixed time update & date display bug with HOC

### PR DESCRIPTION
**Pull request by tmsavage**

### CODE REVIEW

File: app/javascript/components/top-corner-controller.jsx [hunk 1]
```diff
@@ -26,7 +35,13 @@ const Row = styled.div`
...
 `;
 
-const TopCorner = ({ showWeather, cityName, loading, error, location }) => {
+export const TopCorner = ({
+  showWeather,
+  cityName,
+  loading,
+  error,
+  location,
+}) => {
```
Comment (fallynlogan) [inline app/javascript/components/top-corner-controller.jsx:44]: Should this be added in its own refactor commit?

File: app/javascript/components/hocs/with-fragment.jsx [hunk 2]
```diff
@@ -16,6 +23,10 @@ export default function withFragment(WrappedComponent, fragmentObject) {
       };
     }, {});
 
+    if (componentName != null) {
+      console.log('After Data Props: ', fragmentDataProps);
```
Comment (fallynlogan) [inline app/javascript/components/hocs/with-fragment.jsx:27]: What does 'After Data Props' mean?

**Review by fallynlogan — COMMENTED** [review]
LGTM just a few questions!

File: app/javascript/components/top-corner-controller.jsx (hunk 1 above)
Comment (tmsavage) [reply app/javascript/components/top-corner-controller.jsx:44]: I don't think so. This is a very minor "refactor" and is probably not worth a separate commit. But maybe a mentor has another opinion.

File: app/javascript/components/hocs/with-fragment.jsx (hunk 2 above)
Comment (tmsavage) [reply app/javascript/components/hocs/with-fragment.jsx:27]: This statement was just meant to check the differences of props before and after `fragmentDataProps` was computed.

File: app/javascript/components/side-panel.jsx [hunk 3]
```diff
@@ -84,4 +86,4 @@ SidePanel.defaultProps = {
   error: false,
 };
 
-export default withFragment(SidePanel, { location: getSidePanel });
+export default withFragment(SidePanel, { location: getSidePanel }, 'foo');
```
Comment (bencates) [inline app/javascript/components/side-panel.jsx:89]: I think this change was included by mistake.

**Review by bencates — APPROVED** [review]
Pull out that extra `'foo'` and then you're good to go!

**Pull request merged**